src/yoroi-wallets/cardano/nfts.ts

```diff
@@ -0,0 +1,25 @@
+import {getAssetFingerprint} from '../../legacy/format'
+import {CardanoAssetMetadata, NFTMetadata, YoroiNFT} from '../types'
+import {asciiToHex} from '../utils/parsing'
+
+export const convertNft = (assetMetadata: CardanoAssetMetadata, storageUrl: string): YoroiNFT => {
```

Comment:
is this more of a `parseNft`?

Reply:
Not really a parse, as we know the inputs. It is used to convert AssetMetadata to YoroiNFT. Do you want me to move this to api.ts?